jest.config.js

```diff
@@ -7,6 +7,6 @@ module.exports = {
     }
   },
   preset: '@vue/cli-plugin-unit-jest/presets/typescript-and-babel',
-  transformIgnorePatterns: []
-
+  transformIgnorePatterns: [],
+  testTimeout: 30000 // need to extend timeout from default of 5000ms for FileUploadPreview component tests
```

Comment:
I think there's a way to override the timeout for just your test / test file. That would be preferable.

Reply:
ok, yeah that makes sense

Reply:
added timeouts to specific tests